apps/ui/forge.config.js

```diff
@@ -0,0 +1,40 @@
+export default {
+  packagerConfig: {
+    appBundleId: 'com.snapshot.app',
```

Comment:
```suggestion
    appBundleId: 'box.snapshot.app',
```

I think we should use domain we own so either box.snapshot.app or org.snapshot.app